guides/source/action_cable_overview.md

```diff
@@ -142,12 +142,18 @@ established using the following Javascript, which is generated by default in Rai
 
 #### Connect Consumer
 
-```coffeescript
-# app/assets/javascripts/cable.coffee
-#= require action_cable
+```js
+// app/assets/javascripts/cable.js
+//= require action_cable
+//= require_self
+//= require_tree ./channels
+
+(function() {
+  this.App || (this.App = {});
+
+  App.cable = ActionCable.createConsumer();
 
-@App = {}
-App.cable = ActionCable.createConsumer()
+}).call(this);
```

Comment:
This is necessary in order to use `this` in strict mode(If I am not mistaken).  
In addition, CoffeeScript is to use `call (this)` to convert the immediate function, have to match it.